src/main/java/study/racing/step3/Car.java

```diff
@@ -0,0 +1,45 @@
+package study.racing.step3;
+
+import java.util.Objects;
+
+public class Car {
+    private MovingCondition movingCondition;
+    private int distance;
+
+    public Car() {
+        this.distance = 0;
+    }
+
+    public Car(MovingCondition movingCondition) {
+        this.distance = 0;
+        this.movingCondition = movingCondition;
+    }
```

Comment:
```suggestion
    public Car(MovingCondition movingCondition) {
        this(movingCondition, 0)
    }
```
위와 같이 아래 생성자를 활용해서 정의할수도 있어요. [점층적 생성자 패턴](https://kihwan95.tistory.com/m/23) 참고해보셔도 좋을거 같습니다.

다만, Car에 3가지 생성자가 모두 필요할까 싶은데요. 2, 3번째 생성자는 테스트에서만 사용을 하고 있는데 최대한 필요한 생성자만 남기면 어떨까 싶습니다.